scripts/next-data/generateNodeReleasesData.mjs

```diff
@@ -0,0 +1,47 @@
+import nodevu from '@nodevu/core';
+import { writeFile } from 'fs/promises';
+import { join } from 'path';
+
+import { getRelativePath } from './_helpers.mjs';
+
+const __dirname = getRelativePath(import.meta.url);
+const jsonFilePath = join(
+  __dirname,
+  '../../public/static/node-releases-data.json'
```

Comment:
It doesn't need to be inside the /static/ folder, FYI. I rather have it directly on `public/`. `static` is a legacy folder from the current old Node.js Website.

Reply:
I see, thanks for explaining the background.
I will move it.